app/delegate-server/State.purs

```diff
@@ -0,0 +1,182 @@
+module DelegateServer.State
```

Comment:
This module is definitely worth being split up into several ones.

Reply:
Addressed in https://github.com/mlabs-haskell/hydra-auction-offchain/pull/31/commits/d7d194c77dc4ff973cdc0b23040577a35a75f063.